tests/test_authz_authorizer.py

```diff
@@ -106,6 +106,30 @@ def test_validate_fail_one_scope(self):
         assert authorizer.outcome == LIMITED_ALLOW
         assert authorizer.restrictions == {"allow": None, "deny": "self"}
 
+    def test_refs(self):
+        restrictions = {"allow": "*", "deny": "example"}
+        authorizer = self._make_authorizer(
+            {
+                **self.token_payload,
```

Comment:
Maybe remove the `**self.token_payload` and declare the refs explicitly here? (to not jump over the code) This is a little bit magic and may be hard to understand for someone new, but it's just my opinion.

Reply:
`**self.token_payload` is here to add the common fields that are basically always the same in the tests (`exp`, `iss`, `iat`), I'm not sure if declaring them explicitly here makes anything clearer?

Reply:
Allrgiht I prolly missed that refs are defined here in 120.